### ISSUE TITLE
screenshots?

### ISSUE THREAD

**Issue by psifertex** [bowline90/Instruction_Slicer#2]
Some screenshots (or better yet, animated gifs!) would be great to include in the readme/description since they'll show up in the plugin manager UI.

**Comment by c3r34lk1ll3r** [comment]
It can be an idea.

Is it possible to test the plugin.json in the local configuration before the creation of a new release?

**Comment by psifertex** [comment]
Yes, you can test that it's parsed properly using `generate_index.py` from https://github.com/Vector35/community-plugins or you can even use `generate_plugininfo.py` to generate your readme from the json file to validate it.
